pyproject.toml

```
@@ -38,7 +55,7 @@ dependencies = [
     "markdown-it-py (==3.0.0)",
     "markupsafe (==2.0.1)",
     "mdurl (==0.1.2)",
-    "packaging (==24.1)",
+    "packaging (>=23)",
```

```suggestion
    "packaging (==23)",
```
This needs to be an exact version in order to pass the IP-Check. You can specify one, or wait until PR #79 is merged.

---

Solved, left it as it was, no conflicts with #79